.gitattributes

```diff
@@ -20,7 +20,8 @@ reporter/src/funTest/assets/*-expected-*	text eol=lf
 reporter/src/main/resources/**/*.ftl	text eol=lf
 
```

Comment:
i don't understand how this change is relevant to the bug ?!
Why a change to a Reporter test would have an impact on a Scanner test ?

Reply:
I'm not sure what you're referring to, I'm not changing a reporter-test related line in this file. I'm just adding a line for `scanner/src/test/assets/scanoss/filesToScan/*.kt` (and adjusting the indentation in the line above).

Reply:
Yes, it was a mistake from me..